src/styles/single-event-page.ts

```diff
@@ -0,0 +1,124 @@
+import { createStyles } from "@material-ui/core";
+import theme from "./theme";
+import hero from "../resources/img/mantyharju-images/mantyharju-images/hero-image-mantyharju.jpg";
+import { AvRepeat } from "material-ui/svg-icons";
+
+export default createStyles({
+  /**
+   * Mantyharju-theme single event page styles
+   */
+
+   /**
+    * TODO: 
+    * - Improve responsive design for buttons (small screens)
+    * - Post columns styling
+    * - Fonts
+    * 
+    * NOTE: Try using rem instead of em..
+    */
+
+   /**
+    * General front page styles
+    */
+   generalButtonStyle: {
+    paddingTop: "0.4em",
+    paddingBottom: "0.4em",
+    paddingLeft: "1em",
+    paddingRight: "1em",
+    pointerEvents: "auto",
+  },
+
+   /**
+    * Hero-image as background 
+    */
+  heroImageDiv: {
+    backgroundImage: `url(${hero})`,
+    backgroundRepeat: "no-repeat",
+    backgroundPosition: "center center",
+    backgroundSize: "cover",
+    color: "#fff",
+    height:"32vh",
+    maxHeight:"30vw",
```

Comment:
don't use vh (viewport width) as a unit in heights. Instead use vh or percentage or fixed pixels